tests/test_slim_engine.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests for SLiM simulation engine.
+"""
+import sys
+import unittest
+import tempfile
+
+import tskit
+
+import stdpopsim
+import stdpopsim.cli
+from . test_cli import capture_output
+
+
+class TestSLiM_API(unittest.TestCase):
```

Comment:
The ``SLiM`` bits in these class names is redundant -- we're in the test_slim module.